lib/rules/selector-type-no-unknown/index.js

```diff
@@ -89,6 +89,7 @@ const rule = function (actual, options) {
             || svgTags.indexOf(tagNameLowerCase) !== -1
             || keywordSets.nonStandardHtmlTags.has(tagNameLowerCase)
             || mathMLTags.indexOf(tagNameLowerCase) !== -1
+            || tagNameLowerCase === "/deep/"
```

Comment:
@jeddy3 one remarks: remove tests on invalid `/deeeps/` (i think it is not related to this rules).